### ISSUE TITLE
AP:0 and result : 0

### ISSUE THREAD

**Issue by barcahoo** [amdegroot/ssd.pytorch#381]
i have trained my own dataset ,iter 120000,and i modified lower lr=1e_6 ,lower batch_size=8. 
 But loss is still huge ,like 120. and run eval.py ,the result AP:0  , result :0. so i can not detect any class .
Why?  can anybody help me? plz tell me how to solve it,thanks

**Comment by ztianlin** [comment]
I have trained it on COCO2017 with EfficientNet as backbone and I got loss like 7. However, mAP are zeros everywhere...

**Comment by barcahoo** [comment]
@ztianlin  IT is weird ,so have you solved it? how to solve

**Comment by teepakorn123** [comment]
I have the  same problem , Have you solved it?

**Comment by wanglaotou** [comment]
hi, i have came across the same problem, does someone know how to solve it? thanks :)

**Comment by wanglaotou** [comment]
> hi, i have came across the same problem, does someone know how to solve it? thanks :)
i trained the voc07 dataset and eval the test.txt, and i get ap and map likes 0.0002 and 0.0019.
and i trained my own dataset, which also gets very low ap like 0.09.

**Comment by lesonly** [comment]
@barcahoo I have the same problem and This question has puzzled me for a long time. Have you solved it?thanks a lot!

**Comment by barcahoo** [comment]
@lesonly   My problem is the case of dataset names. I changed the lower () function in voc0712. Py. Delete lower() in name = obj. Find ('name '). Text. Lower(). Strip(), or it will be automatically converted to lowercase. Of course, because my dataset categories are all uppercase.

**Comment by lesonly** [comment]
@barcahoo Thank you for your warm reply,Did you eval it on coco dataset?I have trained it on COCO2017,but i got map 0.04

**Comment by barcahoo** [comment]
@lesonly  sorry,i did not. i just used my own dataset,and got map 0.77.if  u are chinese,maybe u can check this website : https://blog.csdn.net/dingkm666/article/details/88775428
and find some issues in the comments below

**Comment by wanglaotou** [comment]
> @barcahoo Thank you for your warm reply,Did you eval it on coco dataset?I have trained it on COCO2017,but i got map 0.04

Hi, may be you can try like this, it works out for me.
![image](https://user-images.githubusercontent.com/12837585/71649766-3ea53400-2d4c-11ea-8c73-02675f8325b9.png)

**Comment by barcahoo** [comment]
@lesonly  sorry,i do not have computer now,and actually i am traveling abroad,sorry

**Comment by lesonly** [comment]
@barcahoo Thank you for your enthusiasm reply.My problem is how to generate the result file to eval on coco dataset.
